internal/scripts/qnx_build_rust_toolchain.sh

```diff
@@ -16,11 +16,10 @@ SCRIPT_START_TIME=$(date +%s)
 # Parse Arguments
 # ---------------
 DRY_RUN=false
-RUST_VERSION="1.88.0"
+RUST_VERSION="1.90.0"
```

Comment:
The new version number needs to be used in the documentation for building the Rust Toolchain as well